.circleci/config.yml

```diff
@@ -206,6 +206,38 @@ jobs:
           command: |
               cd ../lib-ledger-core-build
               ctest
+  generate_doc:
+    docker:
+      - image: circleci/buildpack-deps:stretch
+    steps:
+      - run:
+          command: git clone --depth 1 -b $CIRCLE_BRANCH --recurse-submodules "$CIRCLE_REPOSITORY_URL" .
+          environment:
+            - GIT_SSH_COMMAND: ssh -o StrictHostKeyChecking=no
+      - run:
+          name: Install dependencies
+          command: |
+            sudo apt-get update -q=2
+            sudo apt-get install -qy doxygen libqt5websockets5 libqt5websockets5-dev qt5-default rsync software-properties-common
+            sudo apt-add-repository -u "deb http://httpredir.debian.org/debian stretch-backports main"
```

Comment:
Need `apt-get update` before using ? maybe move it on top

Reply:
>   -u, --update          Update package cache after adding